pkg/network/transports.go

```diff
@@ -49,11 +95,7 @@ func newHTTPTransport(connTimeout time.Duration) http.RoundTripper {
 		ExpectContinueTimeout: 1 * time.Second,
 
 		// This is bespoke.
-		DialContext: (&net.Dialer{
-			Timeout:   connTimeout,
-			KeepAlive: 5 * time.Second,
-			DualStack: true,
-		}).DialContext,
+		DialContext: dialWithBackOff,
```

Comment:
This is awesome, good find.

Reply:
Thanks for finding this.  Last time I look but didn't see :D